Package.resolved

```diff
@@ -77,9 +77,9 @@
         "package": "XcodeProj",
         "repositoryURL": "https://github.com/tuist/XcodeProj.git",
         "state": {
-          "branch": null,
-          "revision": "94e55232d227f9d78b811c98cb2e5d0cbd08987b",
-          "version": "7.22.0"
+          "branch": "main",
```

Comment:
We shouldn't use an unstable version.

This PR may depend on this patch. 
[Allows passing `BuildableIdentifier` String to `BuildableReference` initializer by freddi-kit · Pull Request #605 · tuist/XcodeProj](https://github.com/tuist/XcodeProj/pull/605)

It's better to distribute newer XcodeProj.

Reply:
I wanna use new XcoreProj version with my patch, so I will modify it after new version released. 
That's why this PR is still Draft.